spec/controllers/gardens_controller_spec.rb

```diff
@@ -12,11 +12,72 @@
 
 require 'rails_helper'
 
-describe GardensController do
-  login_member
+RSpec.describe GardensController, type: :controller do
+  include Devise::Test::ControllerHelpers
+  let(:valid_params) { { name: 'My second Garden' } }
 
-  def valid_attributes
-    member = FactoryGirl.create(:member)
-    { name: 'My Garden', owner_id: member.id }
+  context "when not signed in" do
+    let(:garden) { double('garden') }
+    describe 'GET new' do
+      before { get :new, id: garden.to_param }
```

Comment:
It looks like you're consistently using

```
describe "METHOD endpoint" do
   before { make request }
   it { assert something }
end
```

when you could use

```
it "METHOD endpoint" do
    make request
    assert something
end
```

which is what I'm used to seeing - the extra blocks tripped me up on first reading this code. Or am I missing something?